packages/block-library/src/latest-posts/edit.js

```diff
@@ -299,7 +299,7 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								} )
 							}
 						/>
-						<BaseControl className="block-editor-image-alignment-control__row">
+						<BaseControl className="editor-latest-posts-image-alignment-control">
```

Comment:
Curious why you changed the classname here? I don't have a strong opinion since much of this needs a refactor anyway, but it looks like there's some separate CSS that targets this class in https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-posts/style.scss#L77. We might possibly remove that whole chunk.

Reply:
Sorry, I missed that. Class name was too generic, and thought styles were coming from another component.

> Curious why you changed the class name here?

Mainly to make it specific to a block.

Reply:
Done 327f07a.